### PR TITLE
Bring the shape resize handlers to the front

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

File: src/uxbox/ui/workspace/selection.cljs [hunk 1]
```diff
@@ -25,17 +28,97 @@
...
+  (letfn [
+          (on-mouse-down [vid event]
+            (dom/stop-propagation event)
+            (uuc/acquire-action! "ui.shape.resize"
+                                 {:vid vid :shape (:id shape)}))
+
+          (on-mouse-up [vid event]
+            (dom/stop-propagation event)
+            (uuc/release-action! "ui.shape.resize"))]
+    (let [{:keys [x y width height]} (geom/outer-rect-coll [shape])]
```
Comment (niwinz) [inline src/uxbox/ui/workspace/selection.cljs]: Here you can use `geom/outer-rect` for single element.

Comment (jespino) [inline src/uxbox/ui/workspace/selection.cljs]: Fixed

**Comment by jespino** [comment]
I have broken the shapes drag & drop, not merge yet.

**Comment by jespino** [comment]
Ready to merge! :smiley:

**Pull request merged**